libs/pages/layout/src/lib/ui/menu-account/menu-account.tsx

```diff
@@ -0,0 +1,113 @@
+import { SignUp } from 'qovery-typescript-axios'
+import { useNavigate } from 'react-router-dom'
+import { OrganizationEntity } from '@qovery/shared/interfaces'
+import { LOGOUT_URL, ONBOARDING_PROJECT_URL, ONBOARDING_URL, ORGANIZATION_URL } from '@qovery/shared/router'
+import { Avatar, Icon, IconAwesomeEnum, Menu, MenuAlign, MenuDirection } from '@qovery/shared/ui'
+
+export interface MenuAccountProps {
+  organizations: OrganizationEntity[]
+  currentOrganization: OrganizationEntity
+  user: SignUp
+}
+
+export function MenuAccount(props: MenuAccountProps) {
+  const { user, organizations, currentOrganization } = props
+  const navigate = useNavigate()
+
+  const blockOrganization = (organization: OrganizationEntity) => (
+    <div data-testid={`content-${organization.id}`} className="flex items-center">
+      <Icon
+        name={IconAwesomeEnum.CHECK}
+        className={`mr-4 ${currentOrganization.id === organization.id ? 'text-success-500' : 'opacity-0'}`}
+      />
+      <span className="w-8 h-8 rounded-sm flex items-center justify-center mr-3">
+        {organization.logo_url ? (
+          <img className="p-1" src={organization.logo_url} alt="" />
+        ) : (
+          <span className="w-full h-full font-medium text-xs text-text-400 bg-element-light-lighter-400 flex items-center justify-center uppercase rounded-sm">
+            {organization.name.charAt(0)}
+          </span>
+        )}
+      </span>
+      <span className="text-text-700 text-sm font-medium">{organization.name}</span>
+    </div>
+  )
+
+  const menus = [
```

Comment:
Is it possible to type this variable so we can have autocomplete and checked type?